source/help/getting-started/creating-an-account.rst

```diff
@@ -0,0 +1,53 @@
+=================================================
+Creating an account and signing in the first time
+=================================================
+
+It's a relatively straight forward process to create an account on Mattermost and start communicating with other people. You enter some basic information on the account creation page then click **Create Account**.
+
+.. Important::
+  - The account creation process might be different than described here, depending on the policies of your organization.
+  - If your administrator sent you instructions, use those instructions.
+  - If you are not able to create an account using these instructions, contact your administrator.
+
+On the account creation page, you must enter the following information:
+
+email address
+  Enter the address that you want to use for signing in, receiving notifications, resetting passwords, and receiving your account confirmation email.
+user name
+  Enter the text that you want displayed in Mattermost. This is the name that people will know you by. If enabled by your administrator, you can also use this name to sign in.
```

Comment:
I'm not quite sure what "the text you want displayed" means. Perhaps: `Your username appears next to your messages and can be used by other team members to notify you. If enabled by your System Admin, you can use your username to sign in.`